### ISSUE TITLE
Ability to right-click rename bank tags

### ISSUE THREAD

**Issue by Gamer1120** [runelite/runelite#6486]
Short version below.

I use bank tags for everything I do. I have tags for every slayer assignment, every daily... Pretty much, if I ever need to have the same gear at some point in the future, I'll make a tag for it. I've just ran into the following issue. I have a fire_giants tag for Fire Giants. These are usually pretty AFK-able, so I use ranged in the Waterfall dungeon. I now want to slay them using melee in the Catacombs of Kourend. Now, optimally, I'd rename fire_giants to fire_giants_ranged and make a new tag fire_giants_melee, but unfortunately, I have to take out each item from the fire_giants tag, make a new tag fire_giants_ranged, then mass-add them all there by right-clicking the Deposit Inventory button and clicking Tag-inventory fire_giants_ranged. I'd like to see the option to rename a tag on the right-click menu of a tag. This would allow the user to change the name, and all items with that tag would have the old tag removed and the new tag applied to them.

Short version: Right-click rename bank tag tabs.

**Comment by raiyni** [comment]
You can export the tab and edit it in notepad, re copy it, and import it.

**Comment by Gamer1120** [comment]
That's an interesting workaround, I'll use that for now. Thanks for the suggestion!
